src/interface/sparse_pack.hpp

```diff
@@ -142,7 +142,23 @@ class SparsePack : public SparsePackBase {
     // accessed on device via instance of types in the type list Ts...
     // The pack will be created and accessible on the device
     template <class T>
-    SparsePack GetPack(T *pmd, const std::vector<bool> &include_block = {}) const {
+    SparsePack GetPack(T *pmd, std::vector<bool> include_block = {},
+                       bool only_fine_two_level_composite_blocks = true) const {
```

Comment:
Despite the long, descriptive variable name, I am still unsure what this is doing.

Reply:
If a mesh data object corresponds to a `GridType::two_level_composite_mesh`, then if `only_fine_two_level_composite_blocks` is true the pack will only include blocks that are on the finer level of the two.